src/widgets/button.rs

```diff
@@ -61,7 +61,19 @@ pub fn button_render(
 ) -> bool {
     if let Ok((button, styles, mut computed_styles)) = query.get_mut(entity) {
         let hover_color = Color::rgba(0.592, 0.627, 0.749, 1.0); //Color::rgba(0.549, 0.666, 0.933, 1.0);
-                                                                 // let color = Color::rgba(0.254, 0.270, 0.349, 1.0);
+
+        let font_size = if let StyleProp::Value(font_size) = styles.font_size {
```

Comment:
You can use the style prop `resolve_or` function to make this much cleaner.

Reply:
Thank you, much less verbose indeed :)

I made a separate commit. Tell me if you'd prefer to see my changes squashed and I'll rebase my branch.